templates/experiment.html

```diff
@@ -17,21 +17,34 @@
       element.className = element.className.replace("inactive", "primary");
     }
 
-    function isExperimentEnabled() {
-      return AMP.isExperimentOn('{{component}}');
+    function isExperimentEnabled(experimentName) {
+      return AMP.isExperimentOn(experimentName);
     }
 
-    function enableExperiment() {
-      if (!isExperimentEnabled()) {
-        AMP.toggleExperiment('{{component}}');
+    function areAllExperimentsEnabled() {
```

Comment:
I could inline the `AMP.isExperimentOn()` call to remove `isExperimentEnabled()`. Depends what you feel is more readable.